stanford-earth/SE3_BLT
# EARTH-296: Enable callout_filmstrip by default.

# READY FOR REVIEW

# Summary
- Enable stanford_paragraph_callout_filmstrip

# Needed By (Date)
- End of Sprint

# Steps to Test

1. Checkout this branch and the same branch in [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/64) and [stanford_components](https://github.com/SU-SWS/stanford_components/pull/69)
2. config import
3. create complex page with callout filmstrip paragraph type
4. verify it looks correct.

# Associated Issues
- https://github.com/SU-SWS/stanford_paragraph_types/pull/64
- https://github.com/SU-SWS/stanford_components/pull/69

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
